onnxruntime/python/tools/transformers/models/stable_diffusion/benchmark.py

```diff
@@ -464,6 +516,14 @@ def parse_arguments():
         help="Provider to benchmark. Default is CUDAExecutionProvider.",
     )
 
+    parser.add_argument(
+        "-t",
+        "--tuning",
+        action="store_true",
+        help="Enable TunableOp and tuning. "
```

Comment:
extra space in help.

Reply:
Not extra, there are two lines of help str.